onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -189,6 +189,7 @@ def __init__(self, int_max, auto_merge, guess_output_rank, verbose, prefix=''):
         }
         self.aten_op_dispatcher_ = {
             'aten::embedding': self._infer_Gather,
+            'aten::bitwise_or': self._infer_aten_bitwise_or,
```

Comment:
@SherlockNoMad is this (shape inference for the bitwise_or op) necessary ?

Reply:
yes